turbinia/config/turbinia_config.py

```diff
@@ -16,20 +16,57 @@
 from __future__ import unicode_literals
 
 
-"""Dummy Turbinia config file."""
+"""Turbinia Config Template"""
 
-# Turbinia Config
+##########
+# COMMON #
+##########
 
-# 'PSQ' is currently the only valid option
+# Which user account Turbinia runs as
+USER = 'turbinia'
+
+# Turbinia's installion directory
+TURBINIA_DIR = '/opt/turbinia'
+
+# Turbinia source code directory (local git repository)
+SRC_DIR = '%s/src' % TURBINIA_DIR
```

Comment:
I think we should try to find a way to not have a "SRC_DIR" var since in a real production environment we won't run directly from git, but should just have the code installed directly into a specific path.  We can talk more about this one in the meeting too.

Reply:
Right. We do not need both SRC_DIR and TURBINIACTL vars at all. Omitted.